docs/conf/australia/2023/health.rst

```diff
@@ -14,33 +14,7 @@ This policy is up to date as of **November 2022**.
 **Vaccine or Negative COVID-19 Test Requirement**
 -------------------------------------------------
 
-All attendees **must show proof of COVID-19 vaccination OR a negative COVID-19 test** upon first entry to the event. Attendee badges will be used as proof of vaccination or negative test validation throughout the event, each participant will need to pick up their own badge.
-
-**Vaccine Requirement**
-
-If you are showing proof of vaccination to attend in person, you will need to show that you are fully vaccinated against the COVID-19 virus. We require that you:
-
-* have completed a full round of vaccine (1 dose of a single-dose vaccine or 2 doses of a two-dose vaccine)
-* that your most recent dose was at least fourteen days before the conference start
-* that your most recent dose or booster was no more than twelve months before the conference start, or as recommended by your local health authority
-
-The following are acceptable forms of proof of full vaccination:
-
-* Government-issued or government-compliant digital vaccination verification app
-* Physical (or digital photocopy of) COVID-19 vaccination card, record, or certificate issued by a government agency, public health agency, or other authorized vaccine provider (a photo will also suffice).
-
-The proof of vaccination record must show the name of the vaccinated individual along with one other personal identifier (such as date of birth or passport number), name of the manufacturer for vaccine received, and date(s) of vaccination. It must also include the name of the official source issuing the record (e.g., public health agency, government agency, or other authorized vaccine provider).
-
-If you are not sure whether your vaccination certificate is eligible, please email us before you register for the conference and we will check this for you.
-
-**Negative COVID-19 Test** **Requirement**
-
-Instead of proof of vaccination, participants may show proof of a negative COVID-19 test administered by a verified provider, to enter the event.
-
-* Provide a negative COVID-19 test upon entry to the event for the first time.
-* The test must be a PCR or RAT test taken within 72 hours of picking up your badge on-site, or an Antigen test taken within 24 hours.
-
-If you are unable to take a test before the event, we will have a rapid test on-site for you to use to gain entry.
+According to the `current public health guidelines issued by the Victorian Government <https://www.coronavirus.vic.gov.au/information-travellers>`__, there is no vaccination, quarantine, permit, or testing requirement to enter Victoria.
```

Comment:
I feel that this is potentially misleading. Can we split out the Victoria requirement from the conf requirement?

Reply:
I suspect at this level of edit the policy is no longer based on PyCascades and PyCon. @ericholscher ?

Reply:
Updated the health page @plaindocs